apps/nextjs-example/src/utils/standardWallet.ts

```diff
@@ -32,40 +32,93 @@ import {
 } from "@aptos-labs/wallet-standard";
 
 /**
- * A class to create a mock wallet for demonstration a wallet
- * implementation compatible with Aptos AIP-62 Wallet Standard
+ * This class is a template you can modify to implement an AIP-62 Wallet.
```

Comment:
this is a copy of https://github.com/aptos-labs/wallet-standard/blob/main/example/wallet.ts - instead of keep maintaining both docs, can we link to the standard repo here instead?

Reply:
Yep! Added a link to that and updated the docs so they are just documenting the code, not recommending revisions (which is what the template is for).